guides/REAME.md

```diff
@@ -0,0 +1,9 @@
+# Guides
+
+This folder contains documents for how to setup the CI/CD flow for a Knative
+project with the existing tools/scripts provided in this repository.
+
+- [How to setup Prow](./prow_setup.md)
```

Comment:
I think this could be more verbose, like CI/CD flows for a new repo should be here

Reply:
Changed to `How to setup Prow and CI/CD flows for a new repo`